charmcraft/commands/store/__init__.py

```diff
@@ -103,7 +103,17 @@ class LoginCommand(BaseCommand):
     def fill_parser(self, parser):
         """Add own parameters to the general parser."""
         parser.add_argument(
-            "--export", type=pathlib.Path, help="The file to save the credentials to"
+            "--export",
+            type=pathlib.Path,
+            help=(
+                "Export the Charmhub unencrypted secret credentials to a file. "
```

Comment:
I believe we try to keep the help for flags to a single short sentence (this a just a conjecture as I can't find any specifications supporting this).  I think this type of information is more appropriate in the overview.